driver-scala/src/it/scala/org/mongodb/scala/gridfs/GridFSObservableSpec.scala

```diff
@@ -307,6 +307,8 @@ class GridFSObservableSpec extends RequiresMongoDBISpec with FuturesSpec with Be
     val fileAsDocument = filesCollection.find[BsonDocument]().head().futureValue
     fileAsDocument.getDocument("metadata").getBinary("uuid").getType should equal(4.toByte)
     fileAsDocument.getDocument("metadata").getBinary("uuid").asUuid() should equal(uuid)
+
+    client.close()
```

Comment:
If the trait had a loan method that took a settings then we could wrap the close method.  

```
  def withClient(mongoClientSettings: MongoClientSettings, testCode: MongoClient => Any): Unit = {
    val client = MongoClient(mongoClientSettings)
    try testCode(client) // loan the client
    finally {
      client.close()
    }
  }
```

That could be used for `clientEncrypted` as well.

Reply:
I didn't quite understand this.  If you think it should be changed, can you add a commit showing how to do it?

Reply:
Done